### PR TITLE
Add docs and workflow for pusblish them

### PR DESCRIPTION
**Pull request by morenod**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Add docs and CI for publishing them using mk-docs

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Comment by rsevilla87** [comment]
Do you have a preview link?

**Comment by morenod** [comment]
> Do you have a preview link?

https://morenod.github.io/rosa-burner/latest/

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- Create Cluster
+- Delete Cluster
+- Platform Cleanup
+- Watcher
+
+## Available Platforms
+
+Following platforms are available at this moment:
+
+- Rosa
```
Comment (rsevilla87) [inline docs/index.md:28]: This is a bit confusing
image

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
+title: Rosa Burner Docs
+---
+
+## What is rosa-burner
+
+Rosa-burner is a wrapper to automate create-use-destroy of OCP clusters in the different managed platforms.
+
+## How works rosa-burner
```
Comment (rsevilla87) [inline docs/index.md:9]:
```suggestion
## How rosa-burner works
```

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  - platforms/index.md
+  - Rosa:
+    - platforms/rosa/index.md
+    - Terraform: platforms/rosa/terraform.md
+    - Hypershift: platforms/rosa/hypershift.md
+- Measurements: measurements.md
+- Observability & alerting:
+  - observability/index.md
+  - Overview: observability/index.md
+  - Indexing: observability/indexing.md
```
Comment (krishvoor) [inline mkdocs.yml:15]: If we plan on adding content here, please update these to below:

```
- Observability & alerting:
  - docs/observability/index.md
  - Overview: docs/observability/index.md
  - Indexing: docs/observability/indexing.md
```

File: docs/measurements.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+During the installation process, following timers are collected for every cluster installation
+
+- asdf
+- asdf
+- asdfasfd
```
Comment (krishvoor) [inline docs/measurements.md:5]: Can you ptal this?

**Review by krishvoor — CHANGES_REQUESTED** [review]
ptal at the review comments

File: docs/index.md (hunk 1 above)
Comment (morenod) [reply docs/index.md]: modified

File: docs/index.md (hunk 2 above)
Comment (morenod) [reply docs/index.md]: done

File: mkdocs.yml (hunk 3 above)
Comment (morenod) [reply mkdocs.yml]: removed

File: docs/measurements.md (hunk 4 above)
Comment (morenod) [reply docs/measurements.md]: added info on measurements

**Pull request merged**